backend/src/Controller/Admin/CaptainPayment/AdminCaptainPaymentToCompanyController.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace App\Controller\Admin\CaptainPayment;
+
+use App\AutoMapping;
+use App\Controller\BaseController;
+use App\Request\Admin\CaptainPayment\AdminCaptainPaymentCreateRequest;
+use App\Service\Admin\CaptainPayment\AdminCaptainPaymentToCompanyService;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use stdClass;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use OpenApi\Annotations as OA;
+use App\Constant\Captain\CaptainConstant;
+use App\Constant\Main\MainErrorConstant;
+use App\Constant\Payment\PaymentConstant;
+
+/**
+ * Create and fetch Payments from captain.
+ * @Route("v1/admin/captainpaymenttocompany/")
+ */
+class AdminCaptainPaymentToCompanyController extends BaseController
```

Comment:
What is the difference between this controller and AdminCaptainPaymentController?